keys/generate-key-pair.sh

```diff
@@ -136,15 +129,27 @@ fi
 }
 
 function infoKeyCert() {
-#You have to enter information in order to generate a custom self signed certificate as a part of your key pair for SDO Owner Attestation. What you are about to enter is what is called a Distinguished Name or a DN.
-#There are quite a few fields but you can leave some blank. For some fields there will be a default value, If you enter '.', the field will be left blank."
-  : ${countryName:?} ${cityName:?} ${orgName:?} ${emailName:?}
+echo "You have to enter information in order to generate a custom self signed certificate as a part of your key pair for SDO Owner Attestation. What you are about to enter is what is called a Distinguished Name or a DN.
+There are quite a few fields but you can leave some blank. For some fields there will be a default value, If you enter '.', the field will be left blank."
+  echo "Country Name (2 letter code) [AU]:"
+  read countryName
+  echo "State or Province Name (full name) [Some-State]:"
+  read stateName
+  echo "Locality Name (eg, city) []:"
+  read cityName
+  echo "Organization Name (eg, company) [Internet Widgits Pty Ltd]:"
+  read orgName
+  echo "Organizational Unit Name (eg, section) []:"
+  read companyName
+  echo "Common Name (e.g. server FQDN or YOUR name) []:"
+  read yourName
+  echo "Email Address []:"
+  read emailName
   echo '-------------------------------------------------'
 }
 
 #============================MAIN CODE=================================
 
-ensureWeAreUser
 infoKeyCert
 if [[ -n "$keyType" ]] && [[ "$keyType" = "all" ]]; then
   allKeys
```

Comment:
In the output of this script i think you should get rid of the blank lines and lines with all dashes. The linux convention for cmd output is to be succinct whenever possible. Normally dividers like blank lines and dashes are only used for cmd output that is very long.

Also, looks like you need a space between the key type and the word "private". Right now it comes out as "rsaprivate"

And in the last line of output, "has" should be "have": Owner Private Key Tarfile and Owner Public Key have been created